From 4e220a223e462f4cdbaf0a423b674f8ea7e39dff Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 13 Sep 2024 12:17:57 -0700
Subject: [PATCH] Fix TestHebrewCalendarInTemporalLeapYear

---
 source/test/intltest/caltest.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/source/test/intltest/caltest.cpp b/source/test/intltest/caltest.cpp
index 7518f11..f3fe5ed 100644
--- a/source/test/intltest/caltest.cpp
+++ b/source/test/intltest/caltest.cpp
@@ -3998,6 +3998,7 @@ void CalendarTest::TestHebrewCalendarInTemporalLeapYear() {
     for (gc.set(startYear, UCAL_JANUARY, 1);
          gc.get(UCAL_YEAR, status) <= stopYear;
          gc.add(UCAL_DATE, incrementDays, status)) {
+        cal->setTime(gc.getTime(status), status);
         if (failure(status, "add/get/set/getTime/setTime incorrect")) return;
 
         int32_t cal_year = cal->get(UCAL_EXTENDED_YEAR, status);
@@ -4006,6 +4007,7 @@ void CalendarTest::TestHebrewCalendarInTemporalLeapYear() {
             leapTest->set(UCAL_MONTH, 0);
             leapTest->set(UCAL_DATE, 1);
             // If 10 months after TISHRI is TAMUZ, then it is a leap year.
+            leapTest->add(UCAL_MONTH, 10, status);
             hasLeapMonth = leapTest->get(UCAL_MONTH, status) == icu::HebrewCalendar::TAMUZ;
             yearForHasLeapMonth = cal_year;
         }
-- 
2.46.0

